src/dispatch/organization/views.py

```diff
@@ -0,0 +1,104 @@
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException, Query
+from sqlalchemy.orm import Session
+
+from dispatch.database import get_db, search_filter_sort_paginate
+
+from .models import (
+    OrganizationCreate,
+    OrganizationRead,
+    OrganizationUpdate,
+    OrganizationPagination,
+)
+from .service import create, delete, get, get_by_email, update
+
+router = APIRouter()
+
+
+@router.get("/", response_model=OrganizationPagination)
+def get_organizations(
+    db_session: Session = Depends(get_db),
+    page: int = 1,
+    items_per_page: int = Query(5, alias="itemsPerPage"),
+    query_str: str = Query(None, alias="q"),
+    sort_by: List[str] = Query([], alias="sortBy[]"),
+    descending: List[bool] = Query([], alias="descending[]"),
+    fields: List[str] = Query([], alias="fields[]"),
+    ops: List[str] = Query([], alias="ops[]"),
+    values: List[str] = Query([], alias="values[]"),
+):
+    """
+    Get all organization contacts.
+    """
+    return search_filter_sort_paginate(
+        db_session=db_session,
+        model="Organization",
+        query_str=query_str,
+        page=page,
+        items_per_page=items_per_page,
+        sort_by=sort_by,
+        descending=descending,
+        fields=fields,
+        values=values,
+        ops=ops,
+    )
+
+
+@router.post("/", response_model=OrganizationRead)
+def create_organization(
+    *, db_session: Session = Depends(get_db), organization_in: OrganizationCreate
+):
+    """
+    Create a new organization.
+    """
+    organization = get_by_email(db_session=db_session, email=organization_in.name)
```

Comment:
Should this be get_by_name instead? I don't see a column for email in the organization data model. Also get_by_email is not defined in the service module.